astro/src/content/docs/lifecycle/manage-users/verification/gate-accounts-until-user-phone-verified.mdx

```diff
@@ -0,0 +1,181 @@
+---
```

Comment:
I waded into this and forgot that ENG-2023 is responsible for this, but it's probably not bad to leave in here as a starting point.

I think these are different enough that separate pages make sense.

Reply:
From that ticket as of now, the main remaining things that can be done then, that I did NOT do here, are:
1. Explain `identities.verified` and reasons more, including `verifiedInstant`.